src/main/java/brotherman/commands/FindCommand.java

```diff
@@ -0,0 +1,19 @@
+package brotherman.commands;
+
+import brotherman.storage.Storage;
+import brotherman.tasks.TaskList;
+import brotherman.ui.Ui;
+public class FindCommand extends Command {
+
+    private String keyword;
+
+    public FindCommand(String keyword) {
+        super(false);
+        this.keyword = keyword;
+    }
```

Comment:
Considering you have basically 100% class and method javadoc coverage everywhere else (great effort on your part), are the missing javadocs in this class intended?